src/components/Navigation/styles.scss

```diff
@@ -0,0 +1,117 @@
+// Mobile portrait
+.wrapper {
+	width: 100%;
+	height: 100%;
+	background-color: red;
+}
+
+	.button {
+		display: inline-flex;
+		position: fixed;
+		top: 1.15em;
+		right: 1em;
+		align-items: center;
+		font-size: 1.5rem;
+		
+		&:before {
+			content: "";
+			display: block;
+			width: 1em;
+			height: 1.15em;
+			font-size: 1.33em;
+			background: url(images/icons/cross.svg) center / contain no-repeat;
```

Comment:
not for this pass, but in the details pass this `::before` element will likely mutate into a `<span>` with its own `::before` and `::after` so that we can do the fancy cross animation thing - kinda another reason why it'll be good to have its own component; it'll have its own cleverness going on.